dmoj/cptbox/_cptbox.pyx

```diff
@@ -417,25 +423,25 @@ cdef class Process:
         else:
             config.max_fd = 2 + len(fds)
             config.fds = <int*>malloc(sizeof(int) * len(fds))
-            for i in xrange(len(fds)):
+            for i in range(len(fds)):
                 config.fds[i] = fds[i]
 
-        config.debugger_type = self.debugger._debugger_type
-        config.trace_syscalls = self._trace_syscalls
-        config.syscall_whitelist = <bint*>malloc(sizeof(bint) * MAX_SYSCALL_NUMBER)
-        for i in xrange(MAX_SYSCALL_NUMBER):
-            # We have to force exit syscalls to trap, so that we can be notified
-            # that the process spawned successfully when using `seccomp`. Otherwise,
-            # a simple assembly program could terminate without ever trapping.
-            if not self.debugger.is_exit(i):
-                config.syscall_whitelist[i] = self._syscall_whitelist[i]
-            else:
-                config.syscall_whitelist[i] = False
+        if self.use_seccomp:
+            config.avoid_seccomp = False
```

Comment:
It would be nice to have the `config` field also called `use_seccomp` for symmetry (and require less mental remapping)